packages/cascara/src/private/Section/Section.js

```diff
@@ -1,21 +1,23 @@
 import React from 'react';
 import pt from 'prop-types';
+import { Role } from 'reakit/Role';
 
 import styles from './Section.module.scss';
 
 const propTypes = {
+  as: pt.string,
   children: pt.oneOfType([pt.node, pt.arrayOf(pt.node)]),
   footer: pt.oneOfType([pt.node, pt.arrayOf(pt.node)]),
   header: pt.oneOfType([pt.node, pt.arrayOf(pt.node)]),
 };
 
-const Section = ({ children, footer, header }) => {
+const Section = ({ as = 'section', children, footer, header, ...rest }) => {
   return (
-    <section className={styles._}>
+    <Role {...rest} as={as} className={styles._}>
```

Comment:
We are still muting className here. We need `rest.className`

Reply:
Could you confirm this change, please?

Reply:
I think we need sommething like this:

`className={cx(rest.className)}`

Reply:
`<Role {...rest} as={as} className={cx('_', rest.className)}>`
👯